src/hooks/useActionCenterSubscribers/useAppUpdateActionSubscriber.tsx

```diff
@@ -21,10 +21,7 @@ export const useAppUpdateActionSubscriber = () => {
 
   const actionsById = useAppSelector(actionSlice.selectors.selectActionsById)
 
-  const toast = useToast({
-    position: 'bottom-right',
-    duration: isDrawerOpen ? 5000 : null,
-  })
+  const toast = useNotificationToast({ duration: isDrawerOpen ? 5000 : null })
```

Comment:
suggestion: same thing let's not tackle in this PR but `isDrawerOpen` from a context, so we could also extract that within `useNotificationToast()` and have null duration logic hter